tests/worksheets/src/test/scala/tests/worksheets/WorksheetSuite.scala

```diff
@@ -52,6 +52,15 @@ class WorksheetSuite extends BaseSuite {
        |""".stripMargin
   )
 
+  checkDiagnostics(
+    "value-class",
+    """|object Foo {
+       |  case class Bar(b: Int) extends AnyVal
+       |}""".stripMargin,
+    "",
+    modifier = Some("reset-object")
```

Comment:
Is it possible to add multiple modifiers?

Reply:
It is! Added a test to demonstrate it.

Reply:
Awesome